api-reference/10 UI Widgets/dxPopover/1 Configuration/container.md

```diff
@@ -0,0 +1 @@
+The default container is defined during the widget's initialization. It is the viewport. If the viewport is not found, it is the body element. If the viewport and the body element are absent, the **container** is the parent element.
```

Comment:
The default container is defined during the widget's initialization. It is the viewport.
What is "It" referring to? The initialization? If not, clarify and specify